test/https/https.test.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const request = require('request')
+const fs = require('fs')
+const path = require('path')
+const Fastify = require('../..')
+
+// Because we are using a self signed certificate for testing
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
```

Comment:
Sorry, I *always* invert the flag to `rejectUnauthorize`, it's really counterintuitive for me.
Sor `rejectUnauthorize: false` :D.